personalitytest/static/js/src/personalitytest-view.js

```diff
@@ -152,6 +152,7 @@ function PersonalityTestXBlockStudent(runtime, element) {
                             resultDescription.appendChild(categoriesList);
                             resultDescription.appendChild(answersDescritpion);
 
+                            $('.full-result-table', element).empty();
                             $('.full-result-table', element).append(resultDescription);
```

Comment:
why not `$('.full-result-table', element).html(resultDescription)` instead of the two lines?